src/Stripe/Entities/StripeTransfer.cs

```diff
@@ -18,7 +18,7 @@ public class StripeTransfer : StripeObject
         public int Amount { get; set; }
 
         [JsonProperty("application_fee")]
-        public int? ApplicationFee { get; set; }
+        public string ApplicationFee { get; set; }
```

Comment:
You can change this to an expandable property - just look in the codebase for expandable and you'll see how to handle it. This should be renamed ApplicationFeeId so it doesn't confuse anyone - like me.